src/stack/ControllerStackImpl.h

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2013-2017 Nest Labs, Inc.
```

Comment:
is this really a fork from nest labs of 2013?

same for other files (for some reason just noticed it here)